apps/web/app/composables/useEditModeNotification/__tests__/useEditModeNotification.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { mockNuxtImport } from '@nuxt/test-utils/runtime';
+import { useEditModeNotification } from '../useEditModeNotification';
+
+const send = vi.fn();
+
+mockNuxtImport('useNotification', () => () => ({ send }));
+
+describe('useEditModeNotification', () => {
+  let disableActions: Ref<boolean>;
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    disableActions = ref(false);
+  });
+
+  it('should not send notification on mount if disableActions is false', () => {
```

Comment:
```suggestion
  it('should not send notification on mount if editor is in preview mode is false', () => {
```
The name of the variable may change, the concept probably won't.